browser/main/lib/ConfigManager.js

```diff
@@ -87,7 +87,6 @@ function validate (config) {
 }
 
 function _save (config) {
-  console.log(config)
```

Comment:
Should we leave the console.log here? Maybe it will come in handy in the future when the users hit some issue related to the config and they can capture this console.log?  @Rokt33r @sosukesuzuki

Reply:
In general, console.log won't offer much information to users unless they are devs, and if they are devs, they can easily add it back in. In particular, doing something like logging AWS credentials is pretty bad from a security perspective.

Reply:
Yes, clearly logging AWS credentials is a bad idea (maybe it was for debugging purpose have the team forgot to remove it :smile: ) I also agree that console logging the config is a bad idea, but before we can invent some kind of a button to take the current config and attach it to the new issue page, leaving the config in the console for people to take a screenshot of is not a bad idea.

Reply:
@ZeroX-DG I've made the changes you requested!